dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -1441,8 +1645,7 @@ INT_PTR CALLBACK LogOffDialogProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lP
         {
             if (LOWORD(wParam) == WA_INACTIVE)
             {
-                DestroyWindow(hwnd);
-                PostQuitMessage(0);
+                EndDialog(hwnd, 0);
```

Comment:
Perhaps `EndDialog(hwnd, IDCANCEL);` ? (while it's true that here, nobody clicked the `IDCANCEL` button, the dialog effectively quits as if a cancel action occurred.)